src/readme.md

```diff
@@ -0,0 +1,30 @@
+# Rick and morty API
+Challenge from wizeline golang bootcamp 2020
+
+This API fetch data from https://rickandmortyapi.com/ following best practices and clean architecture.
```

Comment:
By convention, you should use bare URLs
Here, you could do `[https://rickandmortyapi.com/]`